tmail-backend/integration-tests/contact-autocomplete/contact-autocomplete-integration-test-common/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Personally I'm not fan of "yet 3 other maven modules". Can't we just put this in TMail JMAP integration tests for now? (Like we did for the encrypted mailbox?)